recipes/jinja2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.8" %}
+
+package:
+  name: jinja2
+  version: {{ version }}
+
+source:
+  fn: Jinja2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
+  md5: edb51693fe22c53cee5403775c71a99e
+
+build:
+  number:  0
+  script: python -m pip install --no-deps .
```

Comment:
Personally, I'd like to stick to the standard.

    python setup.py install --single-version-externally-managed --record record.txt

It is consistent with what we already have and it works great. I'll happily place it somewhere like the example recipe if the only concern is being able to find the arguments.